File: 프로그래머스/lv1/81301. 숫자 문자열과 영단어/숫자 문자열과 영단어.py
def solution(s):
    num_list = ['zero', 'one', 'two', 'three', 'four', 'five', 'six', 'seven', 'eight', 'nine']
    for i, j in enumerate(num_list):
        s = s.replace(j,str(i))
    return int(s)

File: 프로그래머스/lv1/81301. 숫자 문자열과 영단어/숫자 문자열과 영단어.cs
using System;

public class Solution {
    public int solution(string s) {
        foreach (NumList n in  Enum.GetValues(typeof(NumList)))
    s= s.Replace(n.ToString(), ((int)n).ToString());
        return int.Parse(s);
    }
}
public enum NumList
{
    zero = 0,
    one, 
    two,
    three, 
    four, 
    five, 
    six, 
    seven,
    eight, 
    nine
}